pkg/controller/migmigration/restore.go

```diff
@@ -394,6 +396,129 @@ func (t *Task) deleteRestores() error {
 	return nil
 }
 
+// Delete stale Velero Restores in the controller namespace to empty
+// the work queue for next migration.
+func (t *Task) deleteStaleRestores() (int, error) {
+	nDeleted := 0
+	destClient, err := t.getDestinationClient()
```

Comment:
We should probably consider both src and dest here, since a stale restore on the src cluster will still block backups (and vice versa)

Reply:
Addressed.